repo/packages/E/enterprise-cli-subcommands/0/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packagingVersion": "2.0",
```

Comment:
`packagingVersion` needs to be `3.0` if a marathon.json.mustache isn't provided.